### PR TITLE
In Action Update

### PR DESCRIPTION
**Pull request by Katiegilroyy**
New Quotes added to logo page

### CODE REVIEW

File: content/community/action.md [hunk 1]
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
 ---
   <div class="container">
     <div class="row justify-content-center">
+    {{< company name="3M" image="/images/logos/3m.png" link="https://github.com/customer-stories/3m" author_name="Kevin Truckenmiller" author_title="Lead DevOps Engineer in CRSL" >}}
+      We’re moving towards more openness, which ultimately creates a communication culture and a generative culture, rather than one that’s bureaucratic and process-based.
+      {{< /company >}}
+      {{< company name="ADEO" image="/images/logos/adeo.png" link="https://github.com/customer-stories/adeo" author_name="Guilherme Guitee" author_title="InnerSource and Agility Leader" >}}
```
Comment (voborgus) [inline content/community/action.md]: Guilherme Guitte – typo here

File: content/community/action.md [hunk 2]
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
...
+      Our development teams have benefited from the collaborative features of GitHub. It allows them to take co-development to a whole new level by working as a community and maturing our InnerSourcing practices.
       {{< /company >}}
       {{< company name="Vmware" image="/images/logos/vmware.png" link="https://www.youtube.com/watch?v=Y6biezNfJOM" author_name="Minchene Tang" author_title="Vmware" >}}
-        A project isn’t just about the code, there’s a team behind it, including writers, designers and project managers. [InnerSource] gives you the opportunity to collaborate across multiple time zones and business units.
+      A project isn’t just about the code, there’s a team behind it, including writers, designers and project managers. [InnerSource] gives you the opportunity to collaborate across multiple time zones and business units.
+      {{< /company >}}
+      {{< company name="Walmart" image="/images/logos/walmart.png" link="https://resources.github.com/whitepapers/introduction-to-innersource/" author_name="Jeremy King" author_title="Executive Vice President & Chief Technology Officer for Global eCommerce" >}}
```
Comment (voborgus) [inline content/community/action.md]: IMHO, the title is too long and can break the layout. Can we shorten `Chief Technology Officer` -> `CTO`?

File: content/community/action.md [hunk 3]
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Gravity Infosolutions" image="/images/logos/gravity.png" >}}{{< /company >}}
       {{< company name="HBR" image="/images/logos/hbr.png" >}}{{< /company >}}
       {{< company name="HERE Global B.V" image="/images/logos/here.png" >}}{{< /company >}}
+      {{< company name="Huawei" image="/images/logos/huawei.png" >}}{{< /company >}}
       {{< company name="IAG" image="/images/logos/iag.png" >}}{{< /company >}}
       {{< company name="IBM" image="/images/logos/ibm.png" >}}{{< /company >}}
       {{< company name="Intuit" image="/images/logos/intuit.png" >}}{{< /company >}}
+      {{< company name="Wiley" image="/images/logos/wiley.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:167]: Wiley already listed with quote

File: content/community/action.md [hunk 4]
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="US Department of Veteran Affairs" image="/images/logos/Seal_of_the_U.S._Department_of_Veterans_Affairs.svg.png" >}}{{< /company >}}
       {{< company name="Walmart" image="/images/logos/walmart.png" >}}{{< /company >}}
       {{< company name="Wayfair" image="/images/logos/wayfair.png" >}}{{< /company >}}
-      {{< company name="Wiley" image="/images/logos/wiley.png" >}}{{< /company >}}
+      {{< company name="Wipro" image="/images/logos/wipro.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:209]: Wipro already listed with quote

File: content/community/action.md [hunk 5]
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="US Department of Veteran Affairs" image="/images/logos/Seal_of_the_U.S._Department_of_Veterans_Affairs.svg.png" >}}{{< /company >}}
       {{< company name="Walmart" image="/images/logos/walmart.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:207]: Walmart already there at the top

File: content/community/action.md [hunk 6]
```diff
@@ -104,6 +192,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Standard Charted" image="/images/logos/standard.png" >}}{{< /company >}}
       {{< company name="Stripe" image="/images/logos/stripe.png" >}}{{< /company >}}
       {{< company name="Taxfix" image="/images/logos/taxfix.png" >}}{{< /company >}}
+      {{< company name="Tencent" image="/images/logos/tencent.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:197]: Tencent already presented with quote

File: content/community/action.md [hunk 7]
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="US Department of Veteran Affairs" image="/images/logos/Seal_of_the_U.S._Department_of_Veterans_Affairs.svg.png" >}}{{< /company >}}
       {{< company name="Walmart" image="/images/logos/walmart.png" >}}{{< /company >}}
       {{< company name="Wayfair" image="/images/logos/wayfair.png" >}}{{< /company >}}
-      {{< company name="Wiley" image="/images/logos/wiley.png" >}}{{< /company >}}
+      {{< company name="Wipro" image="/images/logos/wipro.png" >}}{{< /company >}}
       {{< company name="Zalando" image="/images/logos/zalando.png" >}}{{< /company >}}
       {{< company name="Zendesk" image="/images/logos/zendesk.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:211]: zendesk appreared with the quote twice

File: content/community/action.md [hunk 8]
```diff
@@ -104,6 +192,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Standard Charted" image="/images/logos/standard.png" >}}{{< /company >}}
       {{< company name="Stripe" image="/images/logos/stripe.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:195]: Stripe already there

File: content/community/action.md [hunk 9]
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
...
-       {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
+      {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
       {{< company name="Nab" image="/images/logos/nab.jpg" >}}{{< /company >}}
       {{< company name="Nationwide" image="/images/logos/nationwide.png" >}}{{< /company >}}
       {{< company name="Navex Global" image="/images/logos/navex-global.png" >}}{{< /company >}}
       {{< company name="Nokia" image="/images/logos/nokia.png" >}}{{< /company >}}
-      {{< company name="Nubank" image="/images/logos/nubank.png" >}}{{< /company >}}
+      {{< company name="Nubanks" image="/images/logos/nubank.png" >}}{{< /company >}}
       {{< company name="Otto Group" image="/images/logos/ottogroup.png" >}}{{< /company >}}
+      {{< company name="PayPal" image="/images/logos/paypal.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:181]: PayPal doubled

File: content/community/action.md [hunk 10]
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
...
+      {{< company name="Microsoft" image="/images/logos/microsoft.png" >}}{{< /company >}}
       {{< company name="Morgan Stanley" image="/images/logos/morganstanley.png" >}}{{< /company >}}
-       {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
+      {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
       {{< company name="Nab" image="/images/logos/nab.jpg" >}}{{< /company >}}
       {{< company name="Nationwide" image="/images/logos/nationwide.png" >}}{{< /company >}}
       {{< company name="Navex Global" image="/images/logos/navex-global.png" >}}{{< /company >}}
       {{< company name="Nokia" image="/images/logos/nokia.png" >}}{{< /company >}}
-      {{< company name="Nubank" image="/images/logos/nubank.png" >}}{{< /company >}}
+      {{< company name="Nubanks" image="/images/logos/nubank.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:179]: Nubanks? It is already with the quote

File: content/community/action.md [hunk 11]
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="US Department of Veteran Affairs" image="/images/logos/Seal_of_the_U.S._Department_of_Veterans_Affairs.svg.png" >}}{{< /company >}}
       {{< company name="Walmart" image="/images/logos/walmart.png" >}}{{< /company >}}
       {{< company name="Wayfair" image="/images/logos/wayfair.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:208]: Listed twice

File: content/community/action.md [hunk 12]
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
...
+      {{< company name="Wiley" image="/images/logos/wiley.png" >}}{{< /company >}}
       {{< company name="Knock" image="/images/logos/knock.png" >}}{{< /company >}}
       {{< company name="Leroy Merlin" image="/images/logos/leroymerlin.png" >}}{{< /company >}}
       {{< company name="Llyods" image="/images/logos/lloyds.png" >}}{{< /company >}}
+      {{< company name="Microsoft" image="/images/logos/microsoft.png" >}}{{< /company >}}
       {{< company name="Morgan Stanley" image="/images/logos/morganstanley.png" >}}{{< /company >}}
-       {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
+      {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
       {{< company name="Nab" image="/images/logos/nab.jpg" >}}{{< /company >}}
       {{< company name="Nationwide" image="/images/logos/nationwide.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:175]: listed twice

File: content/community/action.md [hunk 13]
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
...
+      {{< /company >}}
+      {{< company name="GitHub" image="/images/logos/github.png" link="https://infocus.github.com/sessions/modernizing-and-unifying-development-at-comcast/" author_name="Martin Woodworth" author_title="Director, Developer Relations" >}}
+      The reason why InnerSource works is because like open source you're working with people who are collaborating with different priorities kind of thing because they're working on different things
       {{< /company >}}
       {{< company name="Huawei" image="/images/logos/huawei.png" link="https://www.youtube.com/watch?v=MhNnK_VPLVI" author_name="Willem Jiang and Long Li" author_title="Huawei" >}}
-        We need to build a community around the InnerSource projects of people who share the same ideals.
+      We need to build a community around the InnerSource projects of people who share the same ideals.
+      {{< /company >}}
+      {{< company name="IMB" image="/images/logos/ibm.png" link="https://resources.github.com/whitepapers/introduction-to-innersource/" author_name="Jeff Jagoda" author_title="Senior Site Engineer" >}}
```
Comment (voborgus) [inline content/community/action.md]: typo in the company name

File: content/community/action.md [hunk 14]
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Gravity Infosolutions" image="/images/logos/gravity.png" >}}{{< /company >}}
       {{< company name="HBR" image="/images/logos/hbr.png" >}}{{< /company >}}
       {{< company name="HERE Global B.V" image="/images/logos/here.png" >}}{{< /company >}}
+      {{< company name="Huawei" image="/images/logos/huawei.png" >}}{{< /company >}}
       {{< company name="IAG" image="/images/logos/iag.png" >}}{{< /company >}}
       {{< company name="IBM" image="/images/logos/ibm.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:165]: appeared twice

File: content/community/action.md [hunk 15]
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Gravity Infosolutions" image="/images/logos/gravity.png" >}}{{< /company >}}
       {{< company name="HBR" image="/images/logos/hbr.png" >}}{{< /company >}}
       {{< company name="HERE Global B.V" image="/images/logos/here.png" >}}{{< /company >}}
+      {{< company name="Huawei" image="/images/logos/huawei.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:163]: appearing twice

File: content/community/action.md [hunk 16]
```diff
@@ -50,8 +132,10 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Bloomberg" image="/images/logos/bloomberg.png" >}}{{< /company >}}
       {{< company name="Bosch" image="/images/logos/bosch.png" >}}{{< /company >}}
       {{< company name="BuzzFeed" image="/images/logos/buzzfeed.png" >}}{{< /company >}}
+      {{< company name="Capital One" image="/images/logos/capitalone.png" >}}{{< /company >}}
       {{< company name="CA Technologies" image="/images/logos/CA_Technologies.png" >}}{{< /company >}}
       {{< company name="Caprillary" image="/images/logos/capillary.png" >}}{{< /company >}}
+      {{< company name="Comcast" image="/images/logos/comcast.png" >}}{{< /company >}}
       {{< company name="CHR" image="/images/logos/chr.png" >}}{{< /company >}}
       {{< company name="Continental" image="/images/logos/continental.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:140]: appearing twice

File: content/community/action.md [hunk 17]
```diff
@@ -50,8 +132,10 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Bloomberg" image="/images/logos/bloomberg.png" >}}{{< /company >}}
       {{< company name="Bosch" image="/images/logos/bosch.png" >}}{{< /company >}}
       {{< company name="BuzzFeed" image="/images/logos/buzzfeed.png" >}}{{< /company >}}
+      {{< company name="Capital One" image="/images/logos/capitalone.png" >}}{{< /company >}}
       {{< company name="CA Technologies" image="/images/logos/CA_Technologies.png" >}}{{< /company >}}
       {{< company name="Caprillary" image="/images/logos/capillary.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md]: typo and appearing twice

File: content/community/action.md [hunk 18]
```diff
@@ -50,8 +132,10 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Bloomberg" image="/images/logos/bloomberg.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:132]: already listed with quote

File: content/community/action.md [hunk 19]
```diff
@@ -40,6 +121,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Age of Peers" image="/images/logos/ageofpeerstwo.png" >}}{{< /company >}}
       {{< company name="Ahold Delhaise" image="/images/logos/ahold.png" >}}{{< /company >}}
       {{< company name="American Airlines" image="/images/logos/americanairlines.png" >}}{{< /company >}}
+       {{< company name="ASOS" image="/images/logos/asos.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:124]: already at the top

File: content/community/action.md [hunk 20]
```diff
@@ -40,6 +121,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Age of Peers" image="/images/logos/ageofpeerstwo.png" >}}{{< /company >}}
       {{< company name="Ahold Delhaise" image="/images/logos/ahold.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:122]: appearing twice

File: content/community/action.md [hunk 21]
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
...
       {{< /company >}}
       {{< company name="Wipro" image="/images/logos/wipro.png" link="https://www.youtube.com/watch?v=vApVU26TZGo" author_name="Andrew Aitken" author_title="Wipro" >}}
-        We have thousands of developers working on these projects today and at any given time we’re doing dozens of different projects... 75% of our clients with large global enterprises are trying InnerSource in some way, shape or form.
+      We have thousands of developers working on these projects today and at any given time we’re doing dozens of different projects... 75% of our clients with large global enterprises are trying InnerSource in some way, shape or form.
+      {{< /company >}}
+      {{< company name="Zendesk" image="/images/logos/zendesk.png" link="https://www.youtube.com/watch?v=vApVU26TZGo" author_name="Jason Smale" author_title="VP of Engineering" >}}
+      Our engineering culture is open and centred around teams owning services and being responsible for running them in production.
       {{< /company >}}
       {{< company name="3M" image="/images/logos/3m.png" >}}{{< /company >}}
```
Comment (voborgus) [inline content/community/action.md:117]: company doubled

File: content/community/action.md (hunk 18 above)
Comment (Katiegilroyy) [reply content/community/action.md:132]: All Companies with Quotes have been listed with the Logos, so we can make presentations and keep them together

**Comment by voborgus** [comment]
Thanks for the update. Sorry for the long list of comments :(

File: content/community/action.md (hunk 13 above)
Comment (Katiegilroyy) [reply content/community/action.md]: Changing Typos now, Thank you

**Comment by Katiegilroyy** [comment]
No worries, Thanks for checking!

File: content/community/action.md (hunk 18 above)
Comment (voborgus) [reply content/community/action.md:132]: great! let's click on "resolve" near my comments and check that all companies doubled then (I saw the companies only listed once with quote :))

File: content/community/action.md [hunk 22]
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
...
+      InnerSource paves the way for ADEO to introduce new open source projects and give developers more freedom.
+      {{< /company >}}
+      {{< company name="Ahold Delhaize" image="/images/logos/ahold.png" link="https://github.com/customer-stories/ahold-delhaize" author_name="Joost Hofman" author_title="Head of Tech Enabling" >}}
+      If we give people the right tools and the right platform, it’s a start. We can share more within our company and with each other, growing as an InnerSource organization. 
+      {{< /company >}}
+      {{< company name="ASOS" image="/images/logos/asos.png" link="https://medium.com/asos-techblog/adopting-innersource-at-asos-5618435c8d7" author_name="Rob Bell" author_title="Principal Software Engineer" >}}
+      InnerSource has allowed us to deliver dozens of cross-cutting features more efficiently, spanning multiple teams and services. 
+      {{< /company >}}
+      {{< company name="BBC" image="/images/logos/bbc.png" link="https://www.youtube.com/watch?v=pEGMxe6xz-0" author_name="Steph Egan" author_title="Software Engineering Team Lead" >}}
```
Comment (voborgus) [inline content/community/action.md:21]: Seems there are still companies exists with only 1 occurrence with quote :)

**Comment by voborgus** [comment]
Oops, my mistake! Thanks for your patience :))

**Pull request merged**